app/views/layouts/landing_page.html.slim

```diff
@@ -0,0 +1,50 @@
+doctype html
```

Comment:
does this really need its own unique layout file and opposed to using the default?

Reply:
I didn't know how else to not include things such the links in the header